app/views/users/show.html.erb

```diff
@@ -1 +1,27 @@
+<% content_for :head do %>
+  <%= stylesheet_link_tag 'userstyleshow', media: 'all' %>
+<% end %>
+
 <h1>User Details</h1>
+<h2>Name: <%= @user.name %></h2>
+
+<div class="bio">
+  <h1>Bio:</h1>
+  <%= @user.bio %>
+</div>
+
+<h3>Recent Posts:</h3>
+<ul>
+  <% @user.recent_posts.each do |post| %>
+    <li class="post-list">
+      <strong><%= post.title %></strong>
```

Comment:
- [ ] As required by the last code reviewer 👇🏽 

>  According to the requirements there should be a button on the user show page redirecting to the user post path.

I recommend to use links on the post titles to redirect to the post#show page.